### PR TITLE
Log activation epochs

### PR DESCRIPTION
**Pull request by mateigraura**
Log activation epoch settings. Needed for config refactor  #2879

### CODE REVIEW

File: node/nodeRunner.go [hunk 1]
```diff
@@ -110,6 +112,25 @@ func (nr *nodeRunner) Start() error {
...
+	log.Debug("below signed threshold enable epoch", "epoch", generalSettings.BelowSignedThresholdEnableEpoch)
+	log.Debug("transaction signed with tx hash enable epoch", "epoch", generalSettings.TransactionSignedWithTxHashEnableEpoch)
+	log.Debug("meta protection enable epoch", "epoch", generalSettings.MetaProtectionEnableEpoch)
+	log.Debug("ahead of time gas usage enable epoch", "epoch", generalSettings.AheadOfTimeGasUsageEnableEpoch)
+	log.Debug("gas price modifier enable epoch", "epoch", generalSettings.GasPriceModifierEnableEpoch)
+	log.Debug("repair callback enable epoch", "epoch", generalSettings.RepairCallbackEnableEpoch)
+	log.Debug("max nodes change enable epoch", "epoch", generalSettings.MaxNodesChangeEnableEpoch)
+	log.Debug("block gas and fees re-check enable epoch", "epoch", generalSettings.BlockGasAndFeesReCheckEnableEpoch)
+}
+
```
Comment (bogdan-rosianu) [inline node/nodeRunner.go:133]: please log also enable epochs found in other files, for example, esdt's enable epoch from `systemSmartContractsConfig`

**Review by bogdan-rosianu — DISMISSED** [review]

File: node/nodeRunner.go (hunk 1 above)
Comment (mateigraura) [reply node/nodeRunner.go:147]: done

File: node/nodeRunner.go [hunk 2]
```diff
@@ -94,6 +94,8 @@ func (nr *nodeRunner) Start() error {
 		return err
 	}
 
+	printEnableEpochs(*nr.configs)
```
Comment (SebastianMarian) [inline node/nodeRunner.go:97]: configs instead *nr.configs ?

File: node/nodeRunner.go [hunk 3]
```diff
@@ -110,6 +112,39 @@ func (nr *nodeRunner) Start() error {
 	return nil
 }
 
+func printEnableEpochs(configs config.Configs) {
```
Comment (SebastianMarian) [inline node/nodeRunner.go:115]: configs *config.Configs instead configs config.Configs ?

File: node/nodeRunner.go [hunk 4]
```diff
@@ -110,6 +112,39 @@ func (nr *nodeRunner) Start() error {
...
+	log.Debug("repair callback enable epoch", "epoch", generalSettings.RepairCallbackEnableEpoch)
+	log.Debug("max nodes change enable epoch", "epoch", generalSettings.MaxNodesChangeEnableEpoch)
+	log.Debug("block gas and fees re-check enable epoch", "epoch", generalSettings.BlockGasAndFeesReCheckEnableEpoch)
+
+	systemSCConfig := configs.SystemSCConfig
+
+	log.Debug("staking v2 epoch", "epoch", systemSCConfig.StakingSystemSCConfig.StakingV2Epoch)
+	log.Debug("stake enable epoch", "epoch", systemSCConfig.StakingSystemSCConfig.StakeEnableEpoch)
+	log.Debug("double key protection enable epoch", "epoch", systemSCConfig.StakingSystemSCConfig.DoubleKeyProtectionEnableEpoch)
+
```
Comment (SebastianMarian) [inline node/nodeRunner.go:138]: You can remove empty lines: 138, 140, 142 and 144

File: node/nodeRunner.go (hunk 2 above)
Comment (mateigraura) [reply node/nodeRunner.go]: done

File: node/nodeRunner.go (hunk 3 above)
Comment (mateigraura) [reply node/nodeRunner.go]: done

File: node/nodeRunner.go (hunk 4 above)
Comment (mateigraura) [reply node/nodeRunner.go]: done

**Review by SebastianMarian — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by stefanandy — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed

**Pull request merged**